psi4/unstable.nix

```diff
@@ -0,0 +1,292 @@
+{ lib, buildPythonPackage, buildPackages, makeWrapper, fetchFromGitHub, pkg-config, writeTextFile
```

Comment:
The derivation for unstable is identical to the stable version, expect for one dependency. I suggest to put in one file for now.

Reply:
Done